core/src/main/scala/better/files/UnicodeCharset.scala

```diff
@@ -12,11 +12,12 @@ import scala.collection.JavaConverters._
   * @param writeByteOrderMarkers If set, write BOMs while encoding
   */
 class UnicodeCharset(underlyingCharset: Charset, writeByteOrderMarkers: Boolean)
-    extends Charset(underlyingCharset.name(), underlyingCharset.aliases().asScala.toArray) {
+    extends Charset(underlyingCharset.name(),
```

Comment:
**Pass array into java method**
Pass Array[T] as argument to java method that requires an array. Java arrays have type Array[T | Null].